js/editor.js

```diff
@@ -344,6 +344,16 @@ var Files_Texteditor = {
 		$('#editor_wrap').before(controlBar);
 		this.setFilenameMaxLength();
 		this.bindControlBar();
+		
+		if (file.locked) {
+			$('#editor_controls small.saving-message')
+				.text(t('files_texteditor', 'file is read-only, locked by {locked}', {locked: file.locked}))
+				.show();
+	   	} else if (!file.writeable) {
+			$('#editor_controls small.saving-message')
+				.text(t('files_texteditor', 'file is read-only, edit permission is missing'))
```

Comment:
```suggestion
				.text(t('files_texteditor', 'file is read-only, editing is disabled'))
```

Reply:
@mrow4a do you think my suggested text change is OK?

If so, the text string needs to be adjusted here, and in the acceptance tests.